### PR TITLE
MAINT remove docker build

### LINKED ISSUE #33: MAINT remove docker build

### PR DESCRIPTION
**Pull request by beckermr**
Closes #33

### CODE REVIEW

**Comment by beckermr** [comment]
Man this is weird. I keep getting this error

screen shot 2017-02-28 at 10 40 31 am

**Comment by beckermr** [comment]
@mheilman Well that fixed it, I think. This is weird.

File: setup.py [hunk 1]
```diff
@@ -15,7 +21,8 @@
     url='https://github.com/civisanalytics/muffnn',
     description=('Multilayer Feed-Forward Neural Network (MuFFNN) models with '
                  'TensorFlow and scikit-learn'),
-    long_description=open(os.path.join(THIS_DIR, 'README.md')).read(),
+    long_description=read('README.md'),
     include_package_data=True,
-    license="BSD-3"
+    license="BSD-3",
+    install_requires=read('requirements.txt').split('\n')
```
Comment (mheilman) [inline setup.py]: this [doesn't work in general](http://stackoverflow.com/a/16624700) but seems fine for our requirements.txt

File: muffnn/mlp/tests/test_base.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 import muffnn.mlp.base as base
 
 
-class TestEstimator(base.MLPBaseEstimator):
```
Comment (mheilman) [inline muffnn/mlp/tests/test_base.py]: i think "Test" is more appropriate than "Mock" in this context. it's not really like, say, a mock database connection, simulating complex behavior.  it's just a simple, useless implementation of a base class for testing

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.0.1]
+
+### Changed
+
+- Deprecated docker build.
```
Comment (mheilman) [inline CHANGELOG.md]: "Removed" would be more appropriate than "Deprecated", i think

File: circle.yml [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-    - docker
+  python:
+    version: 3.5.2
 
 dependencies:
-  override:
-    - docker info
-    - docker build -t civisanalytics/muffnn .
+  post:
+    - pip install flake8 pytest pytest-faulthandler
```
Comment (mheilman) [inline circle.yml]: what is this?  could you please add a comment in here about why this is needed?

File: setup.py (hunk 1 above)
Comment (beckermr) [inline setup.py]: Yep. I know. :)

File: muffnn/mlp/tests/test_base.py (hunk 2 above)
Comment (beckermr) [inline muffnn/mlp/tests/test_base.py]: So calling this `TestEstimator` causes pytest to throw a warning, which is why I renamed it.

Comment (mheilman) [inline muffnn/mlp/tests/test_base.py]: `_TestEstimator`?  `SimpleTestEstimator`?

File: circle.yml (hunk 4 above)
Comment (beckermr) [inline circle.yml]: This installs packages needed for dev. See here https://circleci.com/docs/1.0/configuration/

File: muffnn/mlp/tests/test_base.py (hunk 2 above)
Comment (beckermr) [inline muffnn/mlp/tests/test_base.py]: Sure I can give those a try.

**Comment by beckermr** [comment]
@mheilman OK Let's try this again!

**Pull request merged**